docs/book/src/topics/rosa/upgrades.md

```diff
@@ -11,6 +11,8 @@ When the versionGate is set to 'Acknowledge', it will revert to 'WaitForAcknowle
 
 The available upgrades versions for the `ROSAControlPlane` will be listed under `ROSAControlPlane.status.availableUpgrades`
 
+The version channel group `ROSAControlPlane.spec.channelGroup` default to stable. However, it can be set to candidate or nightly. Changing the version channel group will change the `ROSAControlPlane.status.availableUpgrades` accordingly. 
```

Comment:
What controller or actor changes `ROSAControlPlane.status.availableUpgrades`? I don't see that field being changed in this PR.

Reply:
I was actually curious what process changes it rather than the values that it could be.

Is something within the ROSA SaaS setting this value, or some code in this repo?

Reply:
it is in the code [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/controlplane/rosa/controllers/rosacontrolplane_controller.go#L419) , based on the cluster.spec version & channel